extension-androidauto/src/main/java/com/mapbox/maps/extension/androidauto/CompassWidget.kt

```diff
@@ -0,0 +1,28 @@
+package com.mapbox.maps.extension.androidauto
+
+import android.content.Context
+import android.graphics.BitmapFactory
+import com.mapbox.maps.renderer.widget.BitmapWidget
+import com.mapbox.maps.renderer.widget.WidgetPosition
+
+/**
+ * Widget shows compass. Positioned in the top right corner by default.
+ *
+ * @param position position of compass
+ * @param marginX horizontal margin in pixels
+ * @param marginY vertical margin in pixels
+ */
+class CompassWidget(
+  context: Context,
+  position: WidgetPosition = WidgetPosition(
+    horizontal = WidgetPosition.Horizontal.RIGHT,
+    vertical = WidgetPosition.Vertical.TOP,
+  ),
+  marginX: Float = 20f,
+  marginY: Float = 20f,
+) : BitmapWidget(
+  bitmap = BitmapFactory.decodeResource(context.resources, R.drawable.mapbox_compass_icon),
```

Comment:
why not let provide a custom compass bitmap?

Reply:
Well, user can just create `BitmapWidget` from scratch, if all the params are overwritten